runtime/compiler/CMakeLists.txt

```diff
@@ -50,7 +50,8 @@ if(OMR_ARCH_X86)
 	omr_append_flags(CMAKE_ASM_NASM_FLAGS ${asm_inc_dirs})
 endif()
 
-if(OMR_TOOLCONFIG STREQUAL "gnu")
+# osx compilers don't like -static-libgcc
```

Comment:
The comment in the UMA file was:
```
 # Using the linker option -static-libgcc results in an error on OSX. The option -static-libstdc++ is unused. Therefore
    # these options have been excluded from OSX
```

Can you port that over verbatim?

Reply:
fixed